src/main/ml-modules/root/com.marklogic.smart-mastering/process-records.xqy

```diff
@@ -31,5 +31,23 @@ declare function process:process-match-and-merge($uri as xs:string)
  :)
 declare function process:process-match-and-merge($uri as xs:string, $option-name as xs:string)
 {
-  proc-impl:process-match-and-merge($uri, $option-name)
+  proc-impl:process-match-and-merge($uri, $option-name, cts:and-query(()))
+};
+
+(:
+ : Identify matches for a target document. Merge any documents where the match
+ : score is above the merge threshold; record notification for matches above
+ : that threshold.
+ :
+ : @param $uri  URI of the target document
+ : @param $option-name  the name of a set of merge options, which include a reference to a set of match options
+ : @param $filter-query  a cts:query used to filter the matched results
```

Comment:
"a cts:query used to restrict matches to a set, such as a specific entity type or collection"